cfme/common/provider.py

```diff
@@ -310,7 +310,7 @@ def validate_stats(self, ui=False):
                      {'ui': ui},
                      message="do_stats_match_db",
                      num_sec=1000,
-                     delay=60)
+                     delay=120)
```

Comment:
@psav True, but for some reason this solved the issue for CFME 5.7. The test ran fine on 5.6, then it failed on 5.7 due to timeout error. Changing the delay to 120 solved the problem. Maybe 80 would have been a better choice?